### PR TITLE
Add C/Fortran support for Arm Compiler for Linux

### PR DESCRIPTION
**Pull request by willlovett-arm**
Hi folks,

The patch is pretty striaghtfoward (and passess make check) but I wouldn't be surprised if this uncovered issues with armclang++ and/or armflang, since they haven't been enabled before.  Can we try it out on beta?

### CODE REVIEW

**Comment by willlovett-arm** [comment]
Eep - I don't think I intended to merge thousands of commits.  Hopefully my intent (add the HEAD commit to beta) is clear?

**Comment by willlovett-arm** [comment]
Retargeted to master

File: etc/config/fortran.amazon.properties [hunk 1]
```diff
@@ -129,6 +129,15 @@ compiler.fppc64g8.exe=/opt/compiler-explorer/powerpc64/gcc-at12/powerpc64-unknow
 compiler.fppc64g8.name=power64 AT12.0
 compiler.fppc64g8.semver=(snapshot)
 
+###############################
+# ARM HPC
+group.armhpc.compilers=armhpc-ac193a64
```
Comment (partouf) [inline etc/config/fortran.amazon.properties]: &armhpc needs to be added to group.cross.compilers a little bit further up (line 94)

File: etc/config/c.amazon.properties [hunk 2]
```diff
@@ -361,6 +361,15 @@ compiler.cmips564el.exe=/opt/compiler-explorer/mips64el/gcc-5.4.0/mips64el-unkno
 compiler.cmips564el.name=MIPS64 gcc 5.4 (el)
 compiler.cmips564el.semver=5.4
 
+###############################
+# ARM HPC
+group.armhpc.compilers=armhpc-ac193a64
```
Comment (partouf) [inline etc/config/c.amazon.properties]: &armhpc needs to be added to group.ccross.compilers line 202

Comment (willlovett-arm) [reply etc/config/c.amazon.properties]: Thx - done.

**Comment by partouf** [comment]
Thanks!

**Pull request merged**